lib/helpview.gi

```diff
@@ -65,28 +65,26 @@ show := PagerAsHelpViewer,
 );
 
 
-if ARCH_IS_WINDOWS() then
```

Comment:
Instead of unconditionally defining the code below, could we perhaps have a `ARCH_IS_CYGWIN` or so which *does* return `true` here?

Reply:
Ah, forgot that such check exists. Makes perfect sense!